src/Bootstrap/Modal.elm

```diff
@@ -282,6 +283,13 @@ hideOnBackdropClick hide (Config ({ options } as conf)) =
     Config { conf | options = { options | hideOnBackdropClick = hide } }
 
 
+{-| Option to trigger close message when the user clicks on the modal backdrop. Default True.
+-}
+attr : Html.Attribute msg -> Config msg -> Config msg
```

Comment:
Could you change this to attrs (plural) and take a list of html attribute, to keep it in line with other modules.

Reply:
Sure thing :+1: